specification/securityinsights/data-plane/Microsoft.SecurityInsights/preview/2022-12-01-preview/ThreatIntelligence.json

```diff
@@ -51,7 +51,7 @@
           "200": {
             "description": "The API returns 200 when one or many indicators are successfully validated and published.",
             "schema": {
-              "$ref": "#/definitions/UploadIndicatorsResponse"
+              "$ref": "#/definitions/ErrorResponseObject"
```

Comment:
Why are you returning an error response for status code 200 (Success). That does not seem right.

Reply:
Hi @mikekistler , we return empty body if there is no item with validation error. If there are any items failing validation, we will return the validation error in the body, other items (which pass validation) are still processed, hence the 200 status code

Reply:
We will change ErrorResponseObject object to UploadIndicatorsResponse object